service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapperTest.java

```diff
@@ -50,6 +50,8 @@
     private static final String EXPECTED_PLAN_STATEMENT_WITHOUT_PERIOD_END = TEST_DIRECTORY + "expected-plan-statement-7.xml";
     private static final String EXPECTED_PLAN_STATEMENT_WITH_IS_NESTED = TEST_DIRECTORY + "expected-plan-statement-8.xml";
     private static final String INPUT_PROCEDURE_REQUEST_WITHOUT_REQUIRED_AUTHORED_ON = TEST_DIRECTORY + "procedure-request-resource-8.json";
+    private static final String INPUT_PROCEDURE_REQUEST_SINGLE_REASON_CODE = TEST_DIRECTORY + "procedure-request-resource-9.json";
+    private static final String EXPECTED_PROCEDURE_REQUEST_SINGLE_REASON_CODE = TEST_DIRECTORY + "expected-plan-statement-9.xml";
```

Comment:
Now that we have this set of test files, it would be worth renaming `INPUT_PROCEDURE_REQUEST_WITH_REASON_DISPLAY` and `EXPECTED_PLAN_STATEMENT_WITH_REASON_DISPLAY` so that they end with `...WITH_MULTIPLE_REASON_CODES`. It seems that their purpose is now to cover the multiple codes case.